portal/static/portal/sass/styles.scss

```diff
@@ -1321,15 +1324,21 @@ td {
 
 .sub-nav--hidden {
   display: none;
+}
+
+.sub-nav--fixed {
+  display: block;
   margin: 0;
   position: fixed;
   top: 0;
   width: 100%;
 }
 
-.sub-nav--fixed {
+.sub-nav--warning--fixed {
```

Comment:
This one and the one above are the same except for the `top` declaration.  That means you could use the same class for most of it and reduce the duplication.